distributed/tests/test_worker.py

```diff
@@ -147,27 +147,6 @@ def reset(self):
     assert tuple(results) == (3, 7)
 
 
-@pytest.mark.slow
-@gen_cluster()
-async def dont_test_delete_data_with_missing_worker(c, a, b):
-    bad = "127.0.0.1:9001"  # this worker doesn't exist
-    c.who_has["z"].add(bad)
-    c.who_has["z"].add(a.address)
-    c.has_what[bad].add("z")
-    c.has_what[a.address].add("z")
-    a.data["z"] = 5
-
-    cc = rpc(ip=c.ip, port=c.port)
-
-    await cc.delete_data(keys=["z"])  # TODO: this hangs for a while
-    assert "z" not in a.data
-    assert not c.who_has["z"]
-    assert not c.has_what[bad]
-    assert not c.has_what[a.address]
-
-    await cc.close_rpc()
-
-
```

Comment:
This is old garbage. The 'delete_data' does not exist. Note the 'dont_' prefix in the function - this was never executed.